hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/handler/CloseRegionHandler.java

```diff
@@ -52,85 +54,101 @@
   // have a running queue of user regions to close?
   private static final Logger LOG = LoggerFactory.getLogger(CloseRegionHandler.class);
 
-  private final RegionServerServices rsServices;
+  private final RegionServerServices rss;
```

Comment:
Why? `rsServices` was more descriptive.

Reply:
rss is used in a bunch of places. regionServerServices would be better than rsServices but rss is short.

Reply:
I've some around to prefer longer and more verbose over short and terse. But that's just me.